values.yaml

```
@@ -167,7 +167,7 @@ client:
   join: null
 
   # grpc should be set to true if the gRPC listener should be enabled.
-  # This should be set to true if connectInject is enabled.
+  # This will be set to true if connectInject or meshGateway is enabled.
```

```suggestion
  # This will automatically be set to true if connectInject or meshGateway is enabled.
```

This statement isn't entirely true unless we fix it so that connectInject turns this on as well (mentioned in the comment on the client daemonset file).

---

✅